NitroxServer/Serialization/ServerConfig.cs

```diff
@@ -89,7 +89,7 @@ public ServerConfigItem(string itemName, T defaultValue)
                 string text = ConfigurationManager.AppSettings[itemName];
 
                 // Empty string is ignored
-                if (typeof(T) == typeof(string) && (string.IsNullOrEmpty(text) || string.IsNullOrWhiteSpace(text)))
+                if (typeof(T) == typeof(string) && (string.IsNullOrEmpty(text) || string.IsNullOrEmpty(text)))
```

Comment:
`string.IsNullOrEmpty(text) || string.IsNullOrEmpty(text)` Is superfluous. But why not just `if (typeof(T) == typeof(string) && string.IsNullOrWhiteSpace(text))`